### ISSUE TITLE
Is there a way to setup proxies when you're in a corporate network?

### ISSUE THREAD

**Issue by obedmr** [buttercup/buttercup-desktop#1326]
It seems like not having an option to setup proxies when you're behind a corporate network. Any insight would be useful
Thanks

**Comment by perry-mitchell** [comment]
No, not currently.. I'd imagine that Buttercup (Electron+Node) should just use the system's proxy by default but I'm not sure, this would need testing.

Ideally it'd just use the system proxy and then the user configures their proxy in their network settings - this would avoid requiring a user interface, for now. In the future I don't see why we couldn't support a proxy configuration but I'd want that to be a second step.

**Comment by jezzaaa** [comment]
Please, yes!

**Comment by bcarrella** [comment]
My upvote for this enhancement request.
